tests/unit/test_service.py

```diff
@@ -298,7 +298,7 @@ def test_service_description_enable(self):
     def test_service_description_enable_already_enabled(self):
         class AlreadyEnabledResponse:
             status = 409
-            data = '{"status":409,"error":{"code":"service_description_already_enabled"}}'
+            data = '{"status":409,"error":{"code":"service_endpoint_already_added"}}'
```

Comment:
Please revert this change as when adding service descriptions, this "service_endpoint_already_added" does not suit here